### ISSUE TITLE
Button

### ISSUE THREAD

**Issue by dusave** [neutrino-ui/neutrino-ui#2]
**Features**:
[ ] Should allow for component or text content
[ ] Should adopt theme support
[ ] Should pass along any valid HTML Button attributes
[ ] Should have proper a11y support
[ ] Should support light/dark mode
[ ] Should support reduced motion

**Other**:
[ ] Should have unit tests
[ ] Should have full documentation
